models/tasks.go

```diff
@@ -0,0 +1,78 @@
+package models
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+// Task представляет задачу в базе данных.
+type Task struct {
+	ID      string `json:"id"`
+	Date    string `json:"date"`
+	Title   string `json:"title"`
+	Comment string `json:"comment"`
+	Repeat  string `json:"repeat"`
+}
+
+// GetTasks возвращает все задачи из базы данных.
+func GetTasks(db *sql.DB) ([]Task, error) {
+	query := `SELECT id, date, title, comment, repeat FROM scheduler ORDER BY date LIMIT 50`
+	rows, err := db.Query(query)
+	if err != nil {
+		return nil, fmt.Errorf("ошибка выполнения запроса: %v", err)
+	}
+	defer rows.Close()
+
+	var tasks []Task
+	for rows.Next() {
+		var task Task
+		err := rows.Scan(&task.ID, &task.Date, &task.Title, &task.Comment, &task.Repeat)
+		if err != nil {
+			return nil, fmt.Errorf("ошибка сканирования строки: %v", err)
```

Comment:
При оборачивании нужно использовать `%w`
```suggestion
			return nil, fmt.Errorf("ошибка сканирования строки: %w", err)
```